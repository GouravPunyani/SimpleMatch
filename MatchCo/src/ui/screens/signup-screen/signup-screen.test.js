// @flow

import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import Screen from './signup-screen'

test('renders correctly', () => {
  const tree = renderer.create(<Screen />)
  expect(tree).toMatchSnapshot()
})
