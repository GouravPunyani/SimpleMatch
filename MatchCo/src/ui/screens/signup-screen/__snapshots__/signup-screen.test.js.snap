// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": undefined,
      "flex": 1,
      "justifyContent": "flex-start",
      "padding": 16,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#333333",
          "fontFamily": "Nexa",
          "fontSize": 14,
        },
      ]
    }
  >
    Signup screen
  </Text>
</View>
`;
